plugin.yaml

```diff
@@ -421,7 +421,7 @@ node_types:
 
   # distributed logical routers
   cloudify.nsx.dlr:
-    derived_from: cloudify.nodes.Router
+    derived_from: cloudify.nodes.Compute
```

Comment:
I need 3 levels structure:
Compute (as base for all other things, generally it is vm, but for now we can forget about it)
Application server (Something like basic service enable/disable)
Application module (case when we have additional sub nodes like filter in routing rule, that can be changed without recreate full service like routing).